frontend/src/languages/german.js

```diff
@@ -134,6 +134,7 @@ const de = {
     subproject_completion_string: "{0} von {1} erledigt",
     subproject_currency: "German: Sub-project Currency",
     subproject_permissions_title: "German: Set permissions for subproject",
+    subproject_permissions_warning: "Noch eine Berechtigungen zugewiesen für das Teilprojekt",
```

Comment:
eine --> keine?

Reply:
The grammatically correct form would be "Dem Teilprojekt wurden noch keine Berechtigungen zugewiesen"